registrasion/models/inventory.py

```diff
@@ -63,11 +65,13 @@ def __str__(self):
     RENDER_TYPE_RADIO = 1
     RENDER_TYPE_QUANTITY = 2
     RENDER_TYPE_ITEM_QUANTITY = 3
+    RENDER_TYPE_CHECKBOX = 4
 
     CATEGORY_RENDER_TYPES = [
         (RENDER_TYPE_RADIO, _("Radio button")),
         (RENDER_TYPE_QUANTITY, _("Quantity boxes")),
         (RENDER_TYPE_ITEM_QUANTITY, _("Product selector and quantity box")),
+        (RENDER_TYPE_CHECKBOX, _("Checkbox button")),
```

Comment:
This requires a migration; but is otherwise fine. I'll merge this and add the required migration momentarily.